bee-message/src/address/nft.rs

```diff
@@ -1,14 +1,17 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{output::NftId, Error};
+use crate::{output::NftId, util::hex_decode, Error};
 
 use bee_common::packable::{Packable, Read, Write};
 
-use core::{ops::Deref, str::FromStr};
+use core::str::FromStr;
 
 /// A NFT address.
-#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash, derive_more::From)]
+#[derive(
+    Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash, derive_more::From, derive_more::AsRef, derive_more::Deref,
```

Comment:
Very minor nitpick: Should we maybe `use derive_more::{AsRef, DeRef}` to make this shorter? Alternatively, if you want to show that those are special, we could also do: `use derive_more as deriv` (or similar).